src/config.ts

```diff
@@ -174,3 +181,12 @@ export function getStampsConfig({
   // Stamps rewrite is disabled
   return undefined
 }
+
+export function getContentConfig({ BEE_DEBUG_API_URL, REUPLOAD_PERIOD }: EnvironmentVariables = {}):
+  | ContentConfig
+  | undefined {
```

Comment:
You are always returning an object in this function so I believe that `undefined` is not needed.